build/Packaging/AzureSiteExtension/Content/install.ps1

```diff
@@ -187,6 +203,36 @@ function CopyAgentInfo($agentInfoDestination)
 	}
 }
 
+function RemoveNewRelicInstallArtifacts($fromDirectory)
+{
+	WriteToInstallLog "Removing New Relic install artifacts"
+
+	try
+	{
+		if (Test-Path -Path "$fromDirectory\NewRelicPackage")
+		{
+			Remove-Item -Recurse NewRelicPackage -ErrorAction Stop
+		}
+	}
+	catch
+	{
+		WriteToInstallLog "Unable to remove 'NewRelicPackage' directory"
+		exit 1
+	}
+
+	try
+	{
+		if (Test-Path -Path "$fromDirectory\NewRelicCorePackage")
+		{
+			Remove-Item -Recurse NewRelicCorePackage -ErrorAction Stop
+		}
+	}
+	catch {
```

Comment:
Minor nit: make brace style consistent with the previous try/catch

Reply:
👍